### DISCUSSION TITLE
Sync multiple browser windows

### DISCUSSION THREAD

**Discussion by ErikSnoedel** [najdek/f1tv_plus#14] [Ideas]
Is it possible to create the function so sync multiplie browser windows. Like if you watch it on two screens you have two browser screens that you have to sync manualy.

**Comment by najdek** [comment]
It's already an option. Click on "MULTI-VIEW" -> "POPOUTS" -> 2 , and you will have two synchronized windows 😃

**Comment by ErikSnoedel** [comment]
i want one screen with 4 frames and one screen with one frame and then both screens synced. Is this possible?

**Comment by najdek** [comment]
Here you can get a modified script:
https://github.com/najdek/f1tv_plus/raw/c6e076486c7d60786c35d48b730bc6079a1f85f7/f1tv_plus.user.js

It will now open one window with selected amount of frames, and another window with one frame.

This version won't be getting any updates.
I might add this functionality into the "main" version at some point, but I need to think of a clean way to implement this.

**Comment by ErikSnoedel** [comment]
Thanks man. Works perfect now with two screens.

**Comment by ErikSnoedel** [comment]
Could you chance one more thing please. The pop out windows with 1 window need to be the windows that control all feeds. Because this is the window where you watch the main feed on.
